src/infra/ssl/sslCmdRepo.go

```diff
@@ -173,75 +180,119 @@ func (repo *SslCmdRepo) shouldIncludeWww(vhost valueObject.Fqdn) bool {
 	return false
 }
 
-func (repo *SslCmdRepo) ReplaceWithValidSsl(sslPair entity.SslPair) error {
-	path, _ := valueObject.NewMappingPath(infraData.GlobalConfigs.DomainOwnershipValidationUrlPath)
-	matchPattern, _ := valueObject.NewMappingMatchPattern("equals")
-	targetType, _ := valueObject.NewMappingTargetType("inline-html")
-	httpResponseCode, _ := valueObject.NewHttpResponseCode(200)
+func (repo *SslCmdRepo) filterDomainsMappedToServer(
```

Comment:
That's weird, don't you have that information already on the method above? This sound like you're querying the DNS again. Why don't you checked if the domain was pointing to the server on the method above? There you're actually using two DNS servers, here you're using Go net.LookuIP which will use it only the container DNS servers which may fail and there is no fallback.